src/auth.d.ts

```diff
@@ -1174,6 +1174,11 @@ export namespace admin.auth {
      * configuration.
      */
     callbackURL?: string;
+
+    /**
+     * The ability for SAML provider to perform request signing.
+     */
+    enableRequestSigning?: boolean;
```

Comment:
We don't currently support this. I think we should remove it. I think we need the flexibility to have internal implementations not exposed externally.  Many SDKs with auto-generated references already have the ability to set visibility on some APIs and attributes.

Reply:
FWIW, there's no built-in mechanism in TypeScript to hide members of an interface. Any library that's doing so is either using third-party tools or some sort of a hack. So we will have to do the same here (@MathBunny the undocumented `--stripInternal` flag is probably our best bet here).

Reply:
Thanks, I agree. I verified the `@internal` option and it worked as intended (updated the respective PR).